stdlib/urllib/parse.pyi

```diff
@@ -66,7 +66,7 @@ class _NetlocResultMixinBase(Generic[AnyStr]):
 class _NetlocResultMixinStr(_NetlocResultMixinBase[str], _ResultMixinStr): ...
 class _NetlocResultMixinBytes(_NetlocResultMixinBase[bytes], _ResultMixinBytes): ...
 
-class _DefragResultBase(tuple[Any, ...], Generic[AnyStr]):
+class _DefragResultBase(tuple[AnyStr, ...], Generic[AnyStr]):
```

Comment:
```suggestion
class _DefragResultBase(tuple[AnyStr, AnyStr], Generic[AnyStr]):
```
Any reason not to do this?

Reply:
I guess a comment woulddn't go amiss here

Reply:
```suggestion
# Ideally this would be a generic fixed-length tuple,
# but mypy doesn't support that yet: https://github.com/python/mypy/issues/685#issuecomment-992014179
class _DefragResultBase(tuple[AnyStr, ...], Generic[AnyStr]):
```